cicd/forgeops-tests/lib/utils/kubectl.py

```diff
@@ -62,4 +62,4 @@ def __run_cmd_process(cmd):
     response = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
     stdout, stderr = response.communicate()
     assert response.returncode == 0, ' Unexpected return code from Popen() ' + stderr
-    return (stdout.split('\n'), stderr.split('\n'))
+    return stdout.split('\n'), stderr.split('\n')
```

Comment:
I do wonder if long term parsing kubectl commands is going to be harder than using the k8s py libarary. But we can cross that bridge later...

Reply:
There are pro and cons to use the k8s py lib or other libs (like the helm) one.
- pro: you don't have to manipulate output this is manage by the lib, but the cost is not huge from my point of view
- cons: you can't display equivalent command line, this is really useful while debugging tests/deployments => this also allows us to generate shell scripts from the tests, we had that with PyForge and this was really useful for dev to diagnose/investigate an issue